http-codes/index.d.ts

```diff
@@ -0,0 +1,70 @@
+// Type definitions for http-codes 1.0
+// Project: https://github.com/flesler/node-http-codes
+// Definitions by: Mohamed Hegazy <https://github.com/mhegazy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = http_codes;
+declare const http_codes: {
```

Comment:
Instead of `export =`-ing an object, this could have multiple `export const` statements.